src/KernelStrategy.sol

```diff
@@ -59,44 +54,6 @@ contract KernelStrategy is Vault {
     /// @notice Emitted when the hasAllocator flag is set
     event SetHasAllocator(bool hasAllocator);
 
-    /**
-     * @notice Initializes the vault.
-     * @param admin The address of the admin.
-     * @param name The name of the vault.
-     * @param symbol The symbol of the vault.
-     * @param decimals The decimals of the vault.
-     * @param baseWithdrawalFee The base withdrawal fee.
-     * @param countNativeAsset Whether to count the native asset.
-     */
-    function initialize(
```

Comment:
decided to delete this, we only have it for hasAllocators, so why introduce more logic